.gitignore

```diff
@@ -10,3 +10,4 @@ packages/*/dist*
 examples/*/dist*
 **/package
 .sandbox
+packages/cli/generators/datasource/connectors.json
```

Comment:
It's pulled in as part of the `build` process and will be shipped with the appropriate version of the `@loopback/cli` because we run build before a release.

Reply:
If the file is added to `.gitignore`, `npm publish` will skip it by default. Please check.

Reply:
But we have the `dist` folders listed in `.gitignore` and I've verified that they still get published to `npm`.

Reason being `npm` searches for `.gitignore` in each project root and sub-directories ... but for us the `.gitignore` is at the top most level of the monorepo and not in any package hence no files are ignored. 

I also ran `npm pack` within the `cli` package and can verify that `connectors.json` was packed.